src/clone.js

```diff
@@ -1,15 +1,16 @@
 // Type "imports"
+/** @typedef {import("./color.js").default} Color */
 /** @typedef {import("./types.js").PlainColorObject} PlainColorObject */
 
 /**
  *
- * @param {PlainColorObject} color
+ * @param {Color | PlainColorObject} color
```

Comment:
Color implements the `PlainColorObject` so adding `Color` shouldn't be necessary.

Reply:
Right—for some reason I keep forgetting that